server/config.go

```diff
@@ -111,6 +116,7 @@ type Config struct {
 	TranslationsMattermostWebhookURL string
 	TranslationsMattermostMessage    string
 	TranslationsBot                  string
+	TranslationsRepos                []*TranslationsRepo
```

Comment:
The field name and the JSON key in config are different. We need to fix one of these.